examples/common/scripts/vtadmin-down.sh

```diff
@@ -2,8 +2,17 @@
 
```

Comment:
Since we're here, we should also add the license header:
```suggestion

# Copyright 2023 The Vitess Authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
I do not feel comfortable adding a Copyright statement to the file, so I will not commit myself, but feel free to do it yourself if you think it is worth your time.